v4/app/build.gradle

```diff
@@ -18,7 +18,7 @@ android {
         applicationId "exchange.dydx.trading"
         minSdkVersion parent.minSdkVersion
         targetSdkVersion parent.targetSdkVersion
-        versionCode 11
+        versionCode 10000
```

Comment:
Updating this to a large number.  Otherwise if you force mainnet to use DOS's config, the force update screen would show (since DOS's force-update versionCode is > 11).